Form/DoctrineMongoDBTypeGuesser.php

```diff
@@ -147,13 +137,8 @@ public function guessRequired($class, $property)
         return null;
     }
 
-    /**
-     * @param string $class
-     * @param string $property
-     *
-     * @return ValueGuess|null
-     */
-    public function guessMaxLength($class, $property)
+    /** @return ValueGuess|null */
+    public function guessMaxLength(string $class, string $property)
```

Comment:
[`FormTypeGuesserInterface::guessMaxLength()` already has `string` in 5.4](https://github.com/symfony/symfony/blob/5.4/src/Symfony/Component/Form/FormTypeGuesserInterface.php#L38)